Resources/views/Form/markdown_widget.html.twig

```diff
@@ -3,8 +3,8 @@
     <link href="{{ asset('bundles/raphymarkdown/css/markdown.min.css') }}" rel="stylesheet">
     <script src="{{ asset('bundles/raphymarkdown/js/markdown.min.js') }}"></script>
     <script type="text/javascript">
-        var markdown_editor_{{ id }} = new SimpleMDE({
-            element: document.getElementById('{{ id }}')
-        })
+        var config = JSON.parse('{{ mde_config|raw }}');
+        config.element = document.getElementById('{{ id }}');
+        var markdown_editor_{{ id }} = new SimpleMDE(config)
```

Comment:
Hey @Sander-Toonen do you really need this global variable? I removed it in [this pull request](https://github.com/Raphy/RaphyMarkdownBundle/pull/5/files#diff-145cd27ce04bb50356cf3ada11017028R37)